### PR TITLE
update `new_thread_created` text, fixed typo in `new_comment_created`

### PR DESCRIPTION
**Pull request by wrynearson**
~⚠️ **Don't merge until #638 is merged** ⚠️~

Upstream: https://github.com/NASA-IMPACT/nasa-apt/issues/636#issuecomment-1442142744

_I think my linter made the diff seem more different than it really is_

@thenav56, could you address the other issues in the comment ☝️ (email notifications not being sent for resolving and unresolving comment threads)? More info is in [this spreadsheet](https://docs.google.com/spreadsheets/d/1EzsPAZ0dJNjreuEeCh3QlA3DIoFytadUxaydoH8xSds/edit#gid=1371486832&range=A2).

### CODE REVIEW

File: app/email/email_templates.json [hunk 1]
```diff
@@ -1,74 +1,78 @@
...
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user ($role) has bumped the minor vesion number of the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>!</p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "thread_closed": {
+    "subject": "Thread closed",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user ($role) has closed a thread in the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>, section: $section. </p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "thread_reopened": {
+    "subject": "Thread reopened",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user ($role) has reopened a thread in the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>, section: $section. </p><p>Sincerely,</p><p>The APT Team</p>"
+  }
```
Comment (thenav56) [inline app/email/email_templates.json]: @wrynearson Let me know if this needs to be changed.

Comment (wrynearson) [reply app/email/email_templates.json]: @thenav56 yes, that would be great. Please use the new lines 26 and 27 [here](https://docs.google.com/spreadsheets/d/1EzsPAZ0dJNjreuEeCh3QlA3DIoFytadUxaydoH8xSds/edit#gid=260528337&range=A26)

Comment (thenav56) [reply app/email/email_templates.json]: Done
https://github.com/NASA-IMPACT/nasa-apt/blob/735edcc1af318b5f1fbbbc853f4c110417a9e925/app/email/email_templates.json#L70-L77

**Review by thenav56 — APPROVED** [review]

**Comment by wrynearson** [comment]
Hey @thenav56, sorry for the confusion, but I think there's an issue with [line 22 and 23 of the spreadsheet](https://docs.google.com/spreadsheets/d/1EzsPAZ0dJNjreuEeCh3QlA3DIoFytadUxaydoH8xSds/edit#gid=260528337&range=E22). 

I created two new threads as `contributor@apt.com` on [this document](https://nasa-apt.surge.sh/documents/adding-reviewers-to-an-atbd/v1.0):
1. Mentioning `will@developmentseed.org`
2. Not mentioning `will@developmentseed.org`

For both threads, I got the email for line 22, saying that `contributor@apt.com` has added a new comment to an existing thread.

image

image

Did you look at these lines in this PR, or were they not included?

**Comment by wrynearson** [comment]
Also @thenav56, I resolved a thread, but was not notified.

image

**Pull request merged**